neutrino-custom/production.js

```diff
@@ -1,48 +1,26 @@
 'use strict';
-const webpack = require('webpack');
-const basePreset = require('./base');
-const CopyPlugin = require('copy-webpack-plugin');
-const CleanPlugin = require('clean-webpack-plugin');
-const CWD = require('./base').CWD;
-const UI = require('./base').UI;
-const DIST = require('./base').DIST;
-
-// The service domain is used to determine whether login is available in the auth component.
-let SERVICE_DOMAIN = process.env.SERVICE_DOMAIN;
+const { DefinePlugin } = require('webpack');
+const copy = require('neutrino-middleware-copy');
 
 module.exports = neutrino => {
-    basePreset(neutrino);
-
-    neutrino.config.plugin('minify')
-        .inject(BabiliPlugin => new BabiliPlugin({
-            evaluate: false, // prevents some minification errors
-        }
-    ));
-
-    // Define the service domain globally so that window.thServiceDomain can be set:
-    neutrino.config.plugin('define')
-        .use(webpack.DefinePlugin, {
-            SERVICE_DOMAIN: JSON.stringify(SERVICE_DOMAIN)
-        });
-
+    neutrino.config
+        .plugin('minify')
+        // prevents some minification errors
+            .tap(() => [{ evaluate: false }]).end()
+        .plugin('define')
+        // Define the service domain globally so that window.thServiceDomain can be set:
+            .use(DefinePlugin, [{ SERVICE_DOMAIN: JSON.stringify(process.env.SERVICE_DOMAIN) }]);
 
     // The copy plugin is overwritten and not injected so that when this preset is
     // imported in ./local-watch.js and run via `neutrino start`, it is still included
     // in the config (it is only applied in !development by default):
-    neutrino.config.plugin('copy')
-        .use(CopyPlugin, [{
-            context: UI,
-            from: '**'
-        }, {
-            from: './contribute.json',
-            to: 'contribute.json'
-        }], {
-            ignore: ['*.js', '*.jsx', '*.css', '*.html', '*.tmpl',
-                '*.eot', '*.otf', '*.ttf', '*.woff', '*.woff2']
-        });
-
-    // Likewise for this clean plugin:
-    neutrino.config.plugin('clean')
-        .use(CleanPlugin, [DIST], { root: CWD } );
-
+    neutrino.use(copy, {
+        patterns: [
+            { context: neutrino.options.source, from: '**' },
+            { from: './contribute.json', to: 'contribute.json' }
```

Comment:
I've done this in #2321. 
Once that lands, does it mean we can use stock Neutrino behaviour here, and remove this step?